src/cutadapt/help.txt

```diff
@@ -0,0 +1,218 @@
+cutadapt version 4.6
+
+Copyright (C) 2010 Marcel Martin <marcel.martin@scilifelab.se> and contributors
+
+Cutadapt removes adapter sequences from high-throughput sequencing reads.
+
+Usage:
+    cutadapt -a ADAPTER [options] [-o output.fastq] input.fastq
+
+For paired-end reads:
+    cutadapt -a ADAPT1 -A ADAPT2 [options] -o out1.fastq -p out2.fastq in1.fastq in2.fastq
+
+Replace "ADAPTER" with the actual sequence of your 3' adapter. IUPAC wildcard
+characters are supported. All reads from input.fastq will be written to
+output.fastq with the adapter sequence removed. Adapter matching is
+error-tolerant. Multiple adapter sequences can be given (use further -a
+options), but only the best-matching adapter will be removed.
+
+Input may also be in FASTA format. Compressed input and output is supported and
+auto-detected from the file name (.gz, .xz, .bz2). Use the file name '-' for
+standard input/output. Without the -o option, output is sent to standard output.
+
+Citation:
+
+Marcel Martin. Cutadapt removes adapter sequences from high-throughput
+sequencing reads. EMBnet.Journal, 17(1):10-12, May 2011.
+http://dx.doi.org/10.14806/ej.17.1.200
+
+Run "cutadapt --help" to see all command-line options.
+See https://cutadapt.readthedocs.io/ for full documentation.
+
+Options:
+  -h, --help            Show this help message and exit
+  --version             Show version number and exit
+  --debug               Print debug log. Use twice to also print DP matrices
+  -j CORES, --cores CORES
+                        Number of CPU cores to use. Use 0 to auto-detect. Default:
+                        1
```

Comment:
It might still be good to add ${meta_cpus:+--cores $meta_cpus} to the command, in case the user is not using a Docker engine. WDYT?

Reply:
Thanks for this remark, I forgot about the fact that the default for cutadapt is 1.

In [this commit](https://github.com/viash-hub/biobase/pull/7/commits/99b4f0139bfb0584ca8da586a985d6a16ae5e1af) I did the following:

- if `$meta_cpus` is not set, use `--cores 0` which auto-detects the number of available cores
- otherwise, set `--cores $meta_cpus`

You agree, @rcannood?

Reply:
:+1: